client/blocks/followers-count/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External Dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Count from 'components/count';
+import { getSelectedSite } from 'state/ui/selectors';
+
+class FollowersCount extends Component {
```

Comment:
Good point; I've wondered in the past whether it is indeed simpler to start with a function component and refactor when you need state and/or lifecycle methods, or go with the class from the start. In any case I think this is good to 🚢  to get the ball rolling, and then we can update in a new PR — also a pretext to add this to Devdocs? :)